### ISSUE TITLE
Thunderbird 78.1.1 has same install ID as Thunderbird 68.11

### ISSUE THREAD

**Issue by djerius** [flathub/org.mozilla.Thunderbird#130]
My (perhaps incorrect) understanding from https://support.mozilla.org/en-US/kb/dedicated-profile-thunderbird-installation is that different versions of Thunderbird will by default use different profiles. When I installed and ran the 78.1.1 beta (following instructions in #120) I discovered that it was using the profile created by 68.11 (which of course I hadn't backed up).

In both cases the install id (from installs.ini and profiles.ini) is BD520B11F73A6B64. (I ensured that ~/.var/app/org.mozilla.Thunderbird/.thunderbird was empty before the installs)

Is this expected behavior?

Thanks,
Diab

**Comment by Erick555** [comment]
Reading more detailed [profile description](https://support.mozilla.org/en-US/kb/understanding-depth-profile-installation) this isn't about different versions but different installs (the path where app binaries lie must be different):

> Additionally profiles.ini now includes a list of known Firefox installs. **Each new section is named based on a hash of the Firefox installation directory**

So the `BD520B11F73A6B64` is the hash of app installation directory. The installation directory will be always the same in flatpak (`/app/lib/thunderbird/`) regardless of stable/beta branch which means every single flatpak thunderbird user should see `BD520B11F73A6B64` as installation id. I believe the same is truth for installing firefox/thunderbird from linux distro repositories (the hash will be different than one from flatpak but the same for all users and same across different versions).

In conclusion I think it works as expected and I don't see anything we can de about it. Keep in mind that 78.1.1 counts as stable upstream version (unlike beta/nightly) even if they don't recommend to use it.

Also note that it still should prevent you from using current profile (now updated to 78.x ) with TB 68.x which may damage it (but don't try before backing it up first).

**Comment by djerius** [comment]
Thanks for the explanation!
